### PR TITLE
CCP-165: Add some extra logging to investigate duplicate messages

### PR DESCRIPTION
**Pull request by phoebe-lew**
- Send off precautionary 200
- Log slack's event id for each message

### CODE REVIEW

File: skills/hears.js [hunk 1]
```diff
@@ -32,7 +34,8 @@ const learningHandler = async (bot, message) => {
       await transaction(Point.knex(), async trx => {
         const messageRecord = await Message.query(trx).insertAndFetch({
           text: message.event.text,
-          datetime: moment().format(),
+          datetime: message.event.ts,
+          slack_event_id: message.event_id,
```
Comment (ZhangChen199102) [inline skills/hears.js:38]: Maybe also add `slack_event_id` field in `Message` model in `models/schema.js`.

**Review by ZhangChen199102 — CHANGES_REQUESTED** [review]
Nice work, just a minor comment.

File: skills/hears.js (hunk 1 above)
Comment (phoebe-lew) [reply skills/hears.js:38]: good catch, done

**Review by ZhangChen199102 — APPROVED** [review]
👍

**Pull request merged**